_search-plugins/knn/nested-search-knn.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: k-NN search with nested fields
+nav_order: 21
+parent: k-NN search
+grand_parent: Search methods
+has_children: false
+has_math: true
+---
+
+# k-NN search with nested fields
+With k-NN search using nested fields, you can store multiple vectors in a single document and search for the k-nearest documents. 
+Each document is scored using the nearest vector in the nested field. 
```

Comment:
this phase is a bit misleading. It doesn't say if the whole document is scored based on a best score from all the vectors from its nested field(s). Need more details on what is `nearest vector` in the context of this sentence.

Reply:
Right. It could be or could not be the best score from all the vectors from its nested field. Let me rephrase it.